p_sets/2/encryption.rb

```diff
@@ -0,0 +1,43 @@
+module Encryption
+
+  ASCII_LETTERS = {
+    capitals: (65..90),
+    lowercase: (97..122)
+  }
+
+  def cyphertext_reporter(plaintext, integer)
+    puts format('%s', encrypt(plaintext, integer))
```

Comment:
I use `format` when I'm combining strings with other characters that might otherwise get lost in the string interpolation syntax.
```
"#{foo} #{bar}" # OK
"Result for #{foo}: #{bar}" # Less good
format("Result for %s: %s", foo, bar) # Clearer
```

It's still quite subjective, but one advantage of being able to separate the format from the values is they can come from different places - the format string might be a constant, the values might be method arguments.